Makefile

```diff
@@ -150,8 +152,10 @@ vet: ## Run go vet against code.
 ci-lint: golangci-lint
 	$(GOLANGCI_LINT) run --timeout 15m0s
 
+# TODO: add test-python-sdk target
+# Nevertheless, any changes to the python-sdk should first be pushed to the upstream repo.
 .PHONY: test
-test: manifests fmt vet envtest gotestsum test-python-sdk
+test: manifests fmt vet envtest gotestsum
```

Comment:
In my opinion we can create our "test" target in our own Makefile. These changes in here will cause trouble in every rebase. Can we do that?.

Reply:
Yeah, fair.